source/Files/music.json

```diff
@@ -0,0 +1 @@
+{"andre@something.com":{"Childish Gambino":["song1","song2","song3"],"Taylor Swift":["song1","song2","song3"]}}
```

Comment:
@MarcoGutierrez the path to this should be source/files/music. Could you change it and repush?